index.html

```diff
@@ -5,222 +5,219 @@
-                            A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.
-                        </p>
-                        <ul class="stack-2">
-                            <li><button class="html" type="button">HTML</button></li>
-                            <li><button class="bootstrap" type="button">Bootstrap</button></li>
-                            <li><button class="ruby" type="button">Ruby</button></li>
-                        </ul>
-                    </div>
-                    <div class="grid-container">
-                        <div class="grid-item">
+                    </article>
+                    <article class="art-data">
+                        <img class="bg-img2" src="assets/recent_works/bg-img.png" alt="background-image"/>
+                        <div class="overlay-content">
+                            <h3 class="title-post">Professional Art <br> Printing Data</h3>
+                            <p class="description">
+                                A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.
+                            </p>
+                            <ul class="stack-2">
+                                <li><button class="html" type="button">HTML</button></li>
+                                <li><button class="bootstrap" type="button">Bootstrap</button></li>
+                                <li><button class="ruby" type="button">Ruby</button></li>
+                            </ul>
                             <button class="b-button" type="button">See project</button>
                         </div>
-                    </div>
-                </article>
+                    </article>
+                    <article class="art-data">
+                        <img class="bg-img2" src="assets/recent_works/bg-img.png" alt="background-image"/>
+                        <div class="overlay-content">
+                            <h3 class="title-post">Professional Art <br> Printing Data</h3>
+                            <p class="description">
+                                A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.
+                            </p>
+                            <ul class="stack-2">
+                                <li><button class="html" type="button">HTML</button></li>
+                                <li><button class="bootstrap" type="button">Bootstrap</button></li>
+                                <li><button class="ruby" type="button">Ruby</button></li>
+                            </ul>
+                            <button class="b-button" type="button">See project</button>
+                        </div>
+                    </article>
+                </section>
+            </section>
+            <section class="about-content">
```

Comment:
- [ ] 👍 Your "about-content" section is coming along nicely, but there are some differences compared to the design. To improve the section's appearance, please update your code to ensure all elements match the design with proper styling. Pay attention to font styles,  spacing, and any other design-related details to create a more cohesive and visually appealing "about-content" section.
![Frame 48 (3)](https://github.com/allenkays/Portfolio-setup-and-mobile-first/assets/55840999/be21bb72-ad4c-4dbb-9cd6-c85bddef296d)

By making these updates, your section will better reflect the design and enhance the overall aesthetics of your project. Keep up the good work, and I'm excited to see your improvements! 🎨📝🚀